e2e/terraform/packer/windows-2016-amd64/provision.ps1

```diff
@@ -155,6 +157,44 @@ function InstallFromRelease {
     Write-Output "Installed Nomad."
 }
 
+function InstallFromURL {
+    Try {
+        # check that we don't already have this version
+        if (C:\opt\nomad.exe -version `
+          | Select-String -Pattern $nomad_version -SimpleMatch -Quiet) {
+              if (C:\opt\nomad.exe -version `
+                | Select-String -Pattern dev -SimpleMatch -Quiet -NotMatch) {
+                    Write-Output "${nomad_version} already installed"
+                    return
+                }
+          }
+    } Catch {
+        Write-Output "${nomad_version} not previously installed"
+    }
+
+    Stop-Service -Name nomad -ErrorAction Ignore
+
+    Write-Output "Downloading Nomad from: $nomad_url"
+    Try {
+        Remove-Item -Path ./nomad.zip -Force -ErrorAction Ignore
+        Invoke-WebRequest -Uri $nomad_url -Outfile nomad.zip -ErrorAction Stop
+
+        Remove-Item -Path $install_path -Force -ErrorAction Ignore
+        Expand-Archive .\nomad.zip .\ -ErrorAction Stop
+        Remove-Item -Path nomad.zip -Force -ErrorAction Ignore
```

Comment:
Unlike Linux, Windows installer only supports zip files. No tar.gz. I left the Linux one for compatibility purposes, but it may make sense to exclusively use zip files for Linux too.